File: packages/schemas/ts/src/schema-version.ts
import { z } from "zod";

/**
 * Released minors of contract major 1. Writers stamp the minor they targeted;
 * readers must accept every value listed here (see packages/schemas/README.md).
 */
export const SchemaVersionV1 = z.enum(["1.0"]);

export type SchemaVersionV1 = z.infer<typeof SchemaVersionV1>;

File: packages/schemas/rust/src/schema_version.rs
//! Released minors of contract major 1 (see `packages/schemas/README.md`).

use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
pub enum SchemaVersionV1 {
    #[serde(rename = "1.0")]
    V1_0,
}
